File: FClub.Mobile/fclub-mobile-apllication/lib/model/participant.dart
class Participant {
  int? id;
  int? eventId;
  int? memberId;
  String? regisDate;
  int? bonusPoint;
  bool? attendance;
  double? rate;

  Participant(
      {this.id,
      this.eventId,
      this.memberId,
      this.regisDate,
      this.bonusPoint,
      this.attendance,
      this.rate});

  Participant.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    eventId = json['eventId'];
    memberId = json['memberId'];
    regisDate = json['regisDate'];
    bonusPoint = json['bonusPoint'];
    attendance = json['attendance'];
    rate = json['rate'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['eventId'] = this.eventId;
    data['memberId'] = this.memberId;
    data['regisDate'] = this.regisDate;
    data['bonusPoint'] = this.bonusPoint;
    data['attendance'] = this.attendance;
    data['rate'] = this.rate;
    return data;
  }
}

File: FClub.Data/Database/Participant.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

#nullable disable

namespace FClub.Data.Database
{
    [Table("Participant")]
    public partial class Participant
    {
        public Participant()
        {
            EventTickets = new HashSet<EventTicket>();
        }

        [Key]
        [Column("id")]
        public int Id { get; set; }
        [Column("event_id")]
        public int EventId { get; set; }
        [Column("member_id")]
        public int MemberId { get; set; }
        [Column("regis_date", TypeName = "date")]
        public DateTime? RegisDate { get; set; }
        [Column("bonus_point", TypeName = "numeric(19, 5)")]
        public decimal? BonusPoint { get; set; }
        [Column("attendance")]
        public bool? Attendance { get; set; }
        [Column("rate")]
        public double? Rate { get; set; }

        [ForeignKey(nameof(EventId))]
        [InverseProperty(nameof(EventInfo.Participants))]
        public virtual EventInfo Event { get; set; }
        [ForeignKey(nameof(MemberId))]
        [InverseProperty("Participants")]
        public virtual Member Member { get; set; }
        [InverseProperty(nameof(EventTicket.Participant))]
        public virtual ICollection<EventTicket> EventTickets { get; set; }
    }
}
